packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -199,7 +214,9 @@ export default abstract class SqlIntegration
         SELECT
           exposure_query,
           experiment_id,
+          MIN(${experimentNameCol}) as experiment_name,
```

Comment:
You don't need this here.  At this point, the `__experiments` table always has an `experiment_name` column since you are doing that alias above in the `__exposure` queries.

From here to the end of the query, you can just hard-code `experiment_name` and `variation_name` no matter what the value of `hasNameCol` is.

Reply:
Do you mean directly just using experimentNameCol or variationNameCol? Those are technically dependent on hasNameCol right? Cause if hasNameCol is not true, then there are no experiment_name and variation_name columns, so I don't think I could hard code directly experiment_name and variation_name

Reply:
Resolved by new changes